wao/wao_strategy_futures.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime
+from typing import Optional
+from freqtrade.persistence import Trade
+from freqtrade.strategy import IStrategy
+from wao.wao_strategy_controller import WAOStrategyController
+
+
+class WAOStrategy_futures(IStrategy):
+    # Optional order type mapping
+    order_types = {
+        'buy': 'market',
+        'sell': 'market',
+        'stoploss': 'market',
+        'stoploss_on_exchange': False
+    }
+
+    def __init__(self, config: dict, brain, time_out_hours, dup):
+        super().__init__(config)
+        self.controller = WAOStrategyController(brain, time_out_hours, dup)
+
+    def confirm_trade_entry(self, pair: str, order_type: str, amount: float, rate: float,
+                            time_in_force: str, current_time: datetime, entry_tag: Optional[str],
+                            side: str, **kwargs) -> bool:
```

Comment:
There were minor changes here on the inherited WAOStrategies so I have made a separate  WAOStrategy_futures for it but if you want the changes to be on the main WAOStrategy I can revert these changes